proxy/proxy/httpd-conf/spacewalk-proxy-wsgi.conf

```diff
@@ -21,7 +21,7 @@ WSGIPythonPath "/usr/share/rhn"
 </IfVersion>
 
 <LocationMatch "^/*">
-    DirectoryIndex _rhn_proxy/index.html index.html index.htm index.html.var index.shtml index.php index.php4 index.php3 index.phtml index.cgi
+    DirectoryIndex suse_proxy/index.html index.html index.htm index.html.var index.shtml index.php index.php4 index.php3 index.phtml index.cgi
```

Comment:
I think this is wrong. It seems that `suse_proxy` and `uyuni_proxy` only exists on the sources.
After installation, its exists directly in the root dir, right?

I think `suse_proxy/index.html` can be removed

Reply:
Good catch, thanks for pointing it out. Initially I only renamed it, but then I went for the double option removing the subfolder, so yes everything goes in the web root (which is `/srv/www/htdocs`). Fixed now :+1: